.github/stale.yml

```diff
@@ -1,21 +1,49 @@
-# Configuration for stale probot
-# https://probot.github.io/apps/stale/
+# This workflow warns and then closes issues and PRs that have had no activity for a specified amount of time.
 #
-# Number of days of inactivity before an issue becomes stale
-daysUntilStale: 60
-# Number of days of inactivity before a stale issue is closed
-daysUntilClose: 21
-# Issues with these labels will never be considered stale
-exemptLabels:
-  - lifecycle/frozen
-# Label to use when marking an issue as stale
-staleLabel: lifecycle/stale
-# Comment to post when marking an issue as stale. Set to `false` to disable
-markComment: >
-  This issue has been automatically marked as stale because it has not had
-  recent activity. It will be closed if no further activity occurs. Thank you
-  for your contributions.
-# Comment to post when closing a stale issue. Set to `false` to disable
-closeComment: >
-  This issue has been automatically closed because it has not had recent
-  activity. Please comment "/reopen" to reopen it.
\ No newline at end of file
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: '0 0 * * *' # Run every day at midnight
+
+jobs:
+  stale:
+    if: ${{ github.repository == 'kubeflow/pipelines' }}
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+    - uses: actions/stale@v5
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        days-before-stale: 60
+        days-before-close: 21
```

Comment:
I don't currently see a need for a stale bot for GitHub issues. Especially one with such a small timer. 

I would rather the KFP maintainers manage the Issue board and close out irrelevant or stale issues. We are planning to hold regular calls to do issue curation, there are only ~200 issues, and there tend to be a lot of old issues that continue to be relevant.

For Pull request, I don't have a strong opinion, I'm fine with closing stale PRs.